demo/angular-editable-text.js

```diff
@@ -10,31 +10,32 @@
 
 })();
 
-/**
- * Created by Gabriel Grinberg on 6/13/14.
- */
+/*
+   Created by Gabriel Grinberg on 6/13/14.
+   */
 
-(function () {
+(function() {
   'use strict';
-  angular.module('gg.editableText')
-    .directive('editableText', ['EditableTextHelper', function (EditableTextHelper) {
+  angular.module('gg.editableText', ['puElasticInput'])
+    .directive('editableText', ['$timeout', 'EditableTextHelper', function($timeout, EditableTextHelper) {
       return {
         scope: {
           editableText: '=',
           editMode: '=',
           placeholder: '@',
-          onChange: '&'
+          onChange: '&',
+          onReject: '&',
```

Comment:
I think we may want to prefix all of these with something like `et`.  I noticed ui-bootstrap did something similar with an `oib` prefix and the reason is that something as generic as `onChange` or `onSelect` could easily conflict with other installed directives.